### ISSUE TITLE
Will it work again or is it working? 

### ISSUE THREAD

**Issue by Refractario1** [noahmaranesi/Ultimate-Guitar-Hack#8]
Hi, thanks for your work! The project is dead, isn't it? At the time it was downloading somehow until they encrypted the tabs. Any solution?

**Comment by sawyerthemiller** [comment]
Yes they have patched this. I am planning to release a new downloading tool, but it will be closed source in PHP as it took me almost 4 years to find this new method and i don't want to get it patched again...

**Comment by Refractario1** [comment]
Great! I'm interested in buying a copy of that tool!

**Comment by sawyerthemiller** [comment]
> Great! I'm interested in buying a copy of that tool!

Here's the current link. Please do not go crazy downloading tabs as I don't know if that will set off red flags at UG

[TabNab](http://leopardindustries.net/tabnab/)

**Comment by Refractario1** [comment]
The page doesn't seem to be working.

**Comment by sawyerthemiller** [comment]
> The page doesn't seem to be working.

I had to make some changes on the server side which, of course, required background scripting besides PHP. Try it now, but please know, sometimes it may not work due to how it operates

**Comment by BernhardSigl** [comment]
> The page doesn't seem to be working.

same problem.

**Comment by sawyerthemiller** [comment]
without getting too much into the process of how it works (and im not doing this to be mean or anything, but it is incredibly complex and i don’t want to get it taken away again) and hence why this is and will always be free (despite the fact that it's closed source)

sorry for any inconveniences people might have had when it wasnt working. simply put, though, it DOES (usually) work, see [here](https://i.ibb.co/cSJgncmw/Screenshot-2026-06-24-at-10-43-40-pm.png) for proof, well you'll have to trust me the screenshot is of real files lol

the server sided stuff required two internet connected devices, and simply became too much of a hassle for to maintain due to one consistently crashing every 5 days or so (not to mention the extra elecrcity and light pollution in my room at night lol, which made rebooting and setting it back up a pain. so ive decided to just temporarily take it down while i find another way to implement it, and bring it back up with a better abuse prevention system... preferably one that relies on a single device (the server) again sorry for any inconvenience, and ignore the error.htm file in there, it was an old one i copied in from when people from my high school were trying to DDOS my server. fun times.
